### DISCUSSION TITLE
External access

### DISCUSSION THREAD

**Discussion by peterdedecker83** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#737] [Q&A]
When I am in my local LAN and go to the URL/IP of the remote in my browser, everything works fine. However, when I (temporarily) enable port forwarding of port 80 to another port on the external IP of my router, I can access the page of the remote, but it seems like it lost the config. No temperature status is shown, and "hydrojets" appear unlike they were not configured.

![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/44509167/d4fa67a1-7425-42d2-884b-b588071b909f)

**Comment by visualapproach** [comment]
It uses ws on port 81 as well.

**Reply by peterdedecker83** [reply]
OK, thanks. So unless port 81 is forwarded (and accessible) as well, it won't work remote?

**Reply by visualapproach** [reply]
That's right. For mqtt you need another port configurable from the ui. I'm not sure what will happen with the time sync.
